docs/root/start/sandboxes/tls-inspector.rst

```diff
@@ -0,0 +1,100 @@
+.. _install_sandboxes_tls_inspector:
+
+TLS Inspector Listener Filter
+=============================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+	Used to make ``HTTP`` requests.
+
+   :ref:`jq <start_sandboxes_setup_jq>`
+	Parse ``json`` output from the upstream echo servers.
+
+This example demonstrates how the ``TLS``  inspector can be used to select ``FilterChains`` to
+distribute the traffic between upstream clusters according to the matched ``transport_protocol`` and/or
+``application_protocols``.
+
+It also demonstrates the admin statistics generated by the ``TLS`` inspector listener filter.
+
+Step 1: Build the sandbox
+*************************
+
+Change directory to ``examples/tls-inspector`` in the Envoy repository.
```

Comment:
...in the Envoy repository, and bring up the services.

(im aware that the tls docs could also do with being a bit clearer here)